rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -272,7 +271,7 @@
     /**
      * Nodes topology
      */
-    public static TopologyManager topologyManager;
+    public TopologyManager topologyManager;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make topologyManager a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)